numerals/scripts/utils/cldf_zenodo.py

```diff
@@ -34,4 +33,6 @@ def download_from_doi(doi, outdir=pathlib.Path('.')):
                 f['links']['self'],
                 outdir / f['links']['self'].split('/')[-1],
             )
+    sub_dirs = [f for f in outdir.iterdir() if f.is_dir()]
+    assert (outdir / sub_dirs[0] / 'cldf').is_dir(), 'no cldf directory found'
```

Comment:
If this is supposed to be generic, it may be better to look for CLDF datasets using https://pycldf.readthedocs.io/en/latest/dataset.html#dataset-discovery

Reply:
Thanks for the hint